README.md

```diff
@@ -42,42 +42,66 @@ gem install line-bot-api
 
 ## Synopsis
 
```

Comment:
(As you pointed out) To avoid confusion, let's refrain from updating the `README.md` until just before the release. Creating a README-v2.md can help us avoid this issue. :smile: